corehq/apps/hqadmin/management/commands/stale_cases_in_es.py

```diff
@@ -0,0 +1,58 @@
+
+import inspect
+
+from django.core.management.base import BaseCommand
+from datetime import datetime
+from dimagi.utils.chunked import chunked
+
+from casexml.apps.case.models import CommCareCase
+from corehq.apps.es import CaseES
+from corehq.elastic import ES_EXPORT_INSTANCE
+from corehq.util.dates import iso_string_to_datetime
+from corehq.util.couch_helpers import paginate_view
+
+
+class Command(BaseCommand):
+    """
+    Returns list of couch case_ids that are not updated in ES
+
+        $ ./manage.py stale_cases_in_es <DOMAIN> > case_ids.txt
+
+    """
+    help = inspect.cleandoc(__doc__).split('\n')[0]
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+
+    def handle(self, domain, **options):
+        for case_id in get_server_modified_on_for_domain(domain):
+            print(case_id)
```

Comment:
nit: these aren't `case_ids` but three-ples, no?

Reply:
Yes, meant to print as CSV. Will correct, thanks